YouTubeDeepDarkMaterial.user.css

```diff
@@ -2193,6 +2193,59 @@
 		color: var(--dimmer-text) !important;
 	}
 
+	/*Fund raiser*/
+	#header-section.ytd-donation-shelf-renderer,
+	#collapse-controls-section.ytd-donation-shelf-renderer
+	{
+		background-color: var(--hover-background) !important;
+	}
+	#header-text.ytd-donation-shelf-renderer,
+	#collapse-controls-section.ytd-donation-shelf-renderer
+	{
+		color: var(--main-text) !important;
+	}
+	#collapse-controls-section.ytd-donation-shelf-renderer:hover
+	{
+		color: var(--main-color) !important;
+	}		
+	#header-section.ytd-donation-shelf-renderer .style-scope.ytd-donation-shelf-renderer.no-transition
+	{
+		filter: invert(.8);
+	}
+	#donate-section.ytd-donation-shelf-renderer,
+	#creator-messages-section.ytd-donation-shelf-renderer, #nonprofit-section.ytd-donation-shelf-renderer,
+	#nonprofit-title.ytd-donation-shelf-renderer
+	{
+		background-color: var(--second-background);
+		border-color: var(--hover-background);
```

Comment:
The `!important` rule is missing, that's the only reason I had issues. Once added you can merge the PR.

Reply:
I see. Forgot :sweat_smile: